### ISSUE TITLE
Ask for pusher URL during setup

### ISSUE THREAD

**Issue by Floppy** [theodi/bothan#188]
PUSHER_URL is now required in the environment. We need to:

* Add it to the docs for dev mode
* Add it to app.json so that new deploys get it
* Add it to bothan-deploy so that users are prompted for it (unless that happens automatically once it's added to app.json, I dunno)
* Explain where to get one.

**Comment by Floppy** [comment]
This isn't actually right. The pusher URL is set by Heroku itself when it provisions the addon. It should work automatically without any help. Checking it this morning.

**Comment by Floppy** [comment]
But it is needed in dev mode

**Comment by quadrophobiac** [comment]
So which of the original points still apply @Floppy 

* Add it to the docs for dev mode
* ~~Add it to app.json so that new deploys get it~~
* ~~Add it to bothan-deploy so that users are prompted for it (unless that happens automatically once it's added to app.json, I dunno)~~
* Explain where to get one.

^ would above be correct?

**Comment by Floppy** [comment]
yep, that's right - Heroku seems to manage it all automatically, so it's only required for devs.
